dockstore-cli-integration-testing/src/test/java/io/dockstore/client/cli/BaseIT.java

```diff
@@ -50,8 +51,9 @@
  * Base integration test class
  * A default configuration that cleans the database between tests
  */
-@ExtendWith(BaseIT.TestStatus.class)
 @ExtendWith(SystemStubsExtension.class)
+@ExtendWith(MuteForSuccessfulTests.class)
+@ExtendWith(BaseIT.TestStatus.class)
```

Comment:
`BaseIT` redundant here, but probably not worth fixing unless there are other issues